source/components/organisms/Step/Step.js

```diff
@@ -74,7 +74,7 @@ function Step({
         onFieldChange(answers);
       }
       if (updateCaseInContext) {
-        updateCaseInContext(answers, getStatusByType('active:ongoing'), currentPosition);
+        updateCaseInContext(answers, currentPosition);
```

Comment:
Second argument should be the signature object. `currentPosition` should be the 3rd.